### PR TITLE
feature(wc): fds data table

### PR DESCRIPTION
**Pull request by Ping-YUAN**

### CODE REVIEW

File: libs/web-components/data-table/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+### Import
+
+```
+npm i @finastra/data-table
+```
+
+```ts
+import '@finastra/data-table';
+...
+<fds-pie-chart width="380" data='[44, 55, 13, 43, 22]' label="['Apple', 'Mango', 'Orange', 'Watermelon', 'Wiki']"></fds-pie-chart>
```
Comment (bcldvd) [inline libs/web-components/data-table/README.md:24]: fix these

File: libs/web-components/data-table/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+```
+
+### Pure HTML pages
+
+```html
+<script type="module" src="https://unpkg.com/@finastra/charts@latest/dist/src/charts.js?module"></script>
+
+<fds-pie-chart id="pieChart" width="380"></fds-pie-chart>
```
Comment (bcldvd) [inline libs/web-components/data-table/README.md:32]: fix these

File: libs/web-components/data-table/README.md (hunk 1 above)
Comment (Ping-YUAN) [reply libs/web-components/data-table/README.md:24]: aha, forgot to update readme file

Comment (Ping-YUAN) [reply libs/web-components/data-table/README.md:24]: aha, forgot to update readme file

**Review by dorsaffrigui — COMMENTED** [review]
The text in table is not accessible in dark mode, could you use colors from the fds theme? Screenshot 2022-06-13 at 15 00 45

File: libs/web-components/data-table/stories/data-table.stories.ts [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+  },
+  args: {
+    dataSource: ELEMENT_DATA,
+    columns: dataTableColumns,
+    columnsToDisplay: dataTableColumnsToDisplay,
+    selectable: true,
+    multiSelect: false
+  },
+  parameters: {
+    docs: {
```
Comment (dorsaffrigui) [inline libs/web-components/data-table/stories/data-table.stories.ts:223]: Could you add an action in storybook for the DATA_TABLE_EVENTS.DATA_TABLE_ROW_SELECTED event. find an example [here](https://github.com/Finastra/finastra-design-system/blob/19f42883b27bf2c7fc8357580ad52d2e8497d3ec/libs/web-components/stepper/stories/horizontal-stepper.stories.ts#L51)

**Review by dorsaffrigui — APPROVED** [review]

**Review by bcldvd — APPROVED** [review]

**Pull request merged**